Dockerfile.dashboard

```diff
@@ -2,12 +2,12 @@ FROM python:2.7
 
 WORKDIR /code/dashboard
 
-ADD dashboard/requirements_prod.txt /code/dashboard/requirements_prod.txt
-RUN pip install -r requirements_prod.txt
-ADD dashboard/requirements.txt /code/dashboard/requirements.txt
+ADD dashboard/requirements.txt .
 RUN pip install -r requirements.txt
 
-ADD models.py /code/models.py
-ADD settings.py /code/settings.py
-ADD dashboard /code/dashboard
-CMD ["python", "server.py"]
+ADD dashboard .
+ADD models.py .
+ADD settings.py .
```

Comment:
`models.py` and `settings.py` should be added to `/code/`, not to `/code/dashboard`.